src/elements/tiles/Tiles.scss

```diff
@@ -16,6 +16,10 @@ novo-tiles {
         cursor: pointer;
         align-items: center;
 
+        &.disabled {
+            opacity: .5;
```

Comment:
No, I mean if its disabled, then you probably shouldn't want to click on it at all. You have it:

```
        if (this.disabled){
            this.onChange.emit(item.value);
            this.onModelChange(item.value);
            return;
        }
```

which will stilll emit the change event for something that didn't get reflected in the UI.

if you just modify the CSS to have `pointer-events: none` then you won't need the logic above